101-webapp-linux-django/azuredeploy.json

```diff
@@ -0,0 +1,91 @@
+{
+    "contentVersion": "1.0.0.0",
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "parameters": {
+        "webAppName": {
+            "type": "string",
+            "defaultValue": "[concat('Django-', uniqueString(resourceGroup().name, utcNow('F')))]",
+            "metadata": {
+                "description": "That name is the name of our application. It has to be unique.Type a name followed by your resource group name. (<name>-<resourceGroupName>)"
+            }
+        },
+        "location": {
+            "type": "string",
+            "defaultValue": "[resourceGroup().location]",
+            "metadata":{
+                "description": "Location for all resources."
+            }
+        }
+    },
+    "variables": {
+        "sku": "Free",
+        "skuCode": "F1",
+        "workerSize": "0",
+        "appInsights": "[concat(parameters('webAppName'), '-insights')]",
+        "hostingPlanName": "[concat( 'hpn-', resourceGroup().name)]"
+    },
+    "resources": [
+        {
+            "name": "[parameters('webAppName')]",
```

Comment:
see: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#sort-order-of-properties

Reply:
Sorted properties